Our `libzerocash` repository doesn't build.

Running `./get-libsnark` errors out with:

```
g++ -o src/zk_proof_systems/zksnark/ram_zksnark/tests/test_ram_zksnark   src/zk_proof_systems/zksnark/ram_zksnark/tests/test_ram_zksnark.o src/algebra/curves/alt_bn128/alt_bn128_g1.o src/algebra/curves/alt_bn128/alt_bn128_g2.o src/algebra/curves/alt_bn128/alt_bn128_init.o src/algebra/curves/alt_bn128/alt_bn128_pairing.o src/algebra/curves/alt_bn128/alt_bn128_pp.o src/algebra/curves/edwards/edwards_g1.o src/algebra/curves/edwards/edwards_g2.o src/algebra/curves/edwards/edwards_init.o src/algebra/curves/edwards/edwards_pairing.o src/algebra/curves/edwards/edwards_pp.o src/algebra/curves/mnt/mnt4/mnt4_g1.o src/algebra/curves/mnt/mnt4/mnt4_g2.o src/algebra/curves/mnt/mnt4/mnt4_init.o src/algebra/curves/mnt/mnt4/mnt4_pairing.o src/algebra/curves/mnt/mnt4/mnt4_pp.o src/algebra/curves/mnt/mnt46_common.o src/algebra/curves/mnt/mnt6/mnt6_g1.o src/algebra/curves/mnt/mnt6/mnt6_g2.o src/algebra/curves/mnt/mnt6/mnt6_init.o src/algebra/curves/mnt/mnt6/mnt6_pairing.o src/algebra/curves/mnt/mnt6/mnt6_pp.o src/common/data_structures/integer_permutation.o src/common/data_structures/set_commitment.o src/common/default_types/r1cs_ppzkpcd_pp.o src/common/default_types/tinyram_ppzksnark_pp.o src/common/default_types/r1cs_ppzkadsnark_pp.o src/common/default_types/tinyram_zksnark_pp.o src/common/profiling.o src/common/routing_algorithms/as_waksman_routing_algorithm.o src/common/routing_algorithms/benes_routing_algorithm.o src/common/utils.o src/gadgetlib1/constraint_profiling.o src/gadgetlib2/adapters.o src/gadgetlib2/constraint.o src/gadgetlib2/examples/simple_example.o src/gadgetlib2/gadget.o src/gadgetlib2/infrastructure.o src/gadgetlib2/integration.o src/gadgetlib2/pp.o src/gadgetlib2/protoboard.o src/gadgetlib2/variable.o src/relations/circuit_satisfaction_problems/tbcs/examples/tbcs_examples.o src/relations/circuit_satisfaction_problems/tbcs/tbcs.o src/relations/ram_computations/memory/examples/memory_contents_examples.o src/relations/ram_computations/memory/memory_store_trace.o src/relations/ram_computations/memory/ra_memory.o src/relations/ram_computations/rams/fooram/fooram_aux.o src/relations/ram_computations/rams/tinyram/tinyram_aux.o src/algebra/curves/bn128/bn128_g1.o src/algebra/curves/bn128/bn128_g2.o src/algebra/curves/bn128/bn128_gt.o src/algebra/curves/bn128/bn128_init.o src/algebra/curves/bn128/bn128_pairing.o src/algebra/curves/bn128/bn128_pp.o -std=c++11 -Wall -Wextra -Wno-unused-parameter -Wno-comment -Wfatal-errors -O2 -march=native -mtune=native -DUSE_ASM -DMONTGOMERY_OUTPUT -DCURVE_BN128 -Idepinst/include -Isrc -DBN_SUPPORT_SNARK -fPIC -Ldepinst/lib -Wl,-rpath depinst/lib -lgmpxx -lgmp -lboost_program_options -lcrypto -ldl -lz -lzm -lprocps
markdown_py -f README.html README.md -x toc -x extra --noisy
make: markdown_py: Command not found
Makefile:252: recipe for target 'README.html' failed
make: *** [README.html] Error 127
```

(I'm assuming I just have to install `markdown_py` and then I'll hit the next issue)
#307 is blocking on this.

--- a/depends/packages/libzerocash.mk
+++ b/depends/packages/libzerocash.mk
@@ -2,14 +2,14 @@ package=libzerocash
 $(package)_download_path=https://github.com/Electric-Coin-Company/$(package)/archive/
 $(package)_file_name=$(package)-$($(package)_git_commit).tar.gz
 $(package)_download_file=$($(package)_git_commit).tar.gz
-$(package)_sha256_hash=efe7bcace34236b6aba31765a8deb702437cf2f578e8168eb0119d874529165b
-$(package)_git_commit=c64e4af60bca3ffd17e05b0b0aa47b47442be604
+$(package)_sha256_hash=ec9648676389e796f33221ee80a37ca8d448f4de93191c1c0c1c489ac6ee8b3b
+$(package)_git_commit=375e10ee7ec8f807ad1481587f9dcef9ca2b1b06
 
 $(package)_dependencies=libsnark crypto++ openssl boost libgmp
 $(package)_patches=
 
 define $(package)_preprocess_cmds
-  rm libzerocash/allocators.h libzerocash/serialize.h
+  rm libzerocash/allocators.h libzerocash/serialize.h libzerocash/streams.h
 endef
 
 # FIXME: How do we know, at the point where the _build_cms are run, that the

